.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
         - TYPE=EBM
         - TYPE=ACCESS-OM
         - TYPE=ACCESS-CM
+        - TYPE=ACCESS-OM-BGC
+        - TYPE=ACCESS-ESM
```

Comment:
As I thought, you'll need to add
```
- TYPE=ACCESS-OM-BGC
```
to `allow_failures`. I've got getting these to pass down as a TO-DO.

Reply:
??? Isn't that what I did?

Reply:
Sorry for the confusion. Yes you added them, but because `ACCESS-OM-BGC` (like `ACCESS-OM`) can't compile correctly due to missing libraries you need to add it to `allowed_failures` to get a clean build test, so when other tests fail, which shouldn't, it will be noticed.

Reply:
So add
```
- TYPE=ACCESS-OM-BGC
```
after here
https://github.com/mom-ocean/MOM5/blob/master/.travis.yml#L29

Reply:
Sorry to be a pain. I've made the change so you don't have to.